README.md

```diff
@@ -51,6 +51,11 @@ Building instructions
 `unpaper` uses GNU Autotools for its build system, so you should be
 able to execute the same commands used for other software packages:
 
+    libtoolize --force
+    aclocal
+    autoheader
+    automake --force-missing --add-missing
+    autoconf
```

Comment:
This should actually be `autoreconf -i` (and only when building out of the Git repository).